### DISCUSSION TITLE
Picker.Cascader的省市级联数据

### DISCUSSION THREAD

**Discussion by jkiss** [ant-design/ant-design-mobile#3889] [Show and tell]
最近在做一个React SPA的个人中心里面的一个修改地区的功能，经过多番寻找，最终决定用ant-mobile的Picker，顺带整理出一个[省市级联数据](https://github.com/jkiss/china-map-data/tree/master/Ant-Mobile/Cascader%E7%9C%81%E5%B8%82%E6%95%B0%E6%8D%AE)，分为二层级联和三层级联2个数据文件，如需自取:zany_face:

**Comment by xiaohuoni** [comment]
项目中一直用这个[@bang88/china-city-data](https://github.com/bang88/china_city_data)，还持续更新。可以试试，结构符合antd-mobile 和 antd-mobile-rn.

**Reply by jkiss** [reply]
cool
